test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -1022,6 +1022,90 @@ public void TryReadAsBool_ReturnsExpectedBoolValue(object value, bool expectedVa
             Assert.Equal(resultValue, expectedValueResult);
         }
 
+        [Fact]
+        public void ValidateRetryOptions_ThrowsWhenMaxRetryIsNull()
+        {
+            var retryOptions = new RetryOptions
+            {
+                Strategy = RetryStrategy.FixedDelay
+            };
+
+            Assert.False(IsValidateRetryOptionsSuccess(retryOptions));
+        }
+
+        [Fact]
+        public void ValidateRetryOptions_ThrowsWhenFixedDelayArgsNull()
+        {
+            var retryOptions = new RetryOptions
+            {
+                Strategy = RetryStrategy.FixedDelay,
+                MaxRetryCount = 5
+            };
+
+            Assert.False(IsValidateRetryOptionsSuccess(retryOptions));
```

Comment:
I see that the "IsValidateRetryOptionsSuccess" test helper catches all exceptions and returns false. I think it might be even better to use "Assert.Throws" in these tests to be more specific when validating the outcome of the method. 

Something like this:
```csharp
var ex = Assert.Throws<ArgumentNullException>(() => Utility.ValidateRetryOptions(retryOptions));
Assert.Equal("Value cannot be null. (Parameter 'DelayInterval').", ex.Message);
```